wgs/wgs-gather-vcfs/wgs-gather-vcfs.wdl

```diff
@@ -0,0 +1,169 @@
+## wgs-gather-vcfs.wdl
+## 
+## This workflow runs the GATK's `GatherVcfsCloud` on a GCS directory containing sharded VCF files
+## from a joint genotyping workflow such as the one here: https://github.com/gatk-workflows/gatk4-germline-snps-indels.
+##
+## Each VCF file corresponds to an interval from the joint genotyping intervals file. That file is a required
+## input to this workflow.
+##
+## The output of the workflow is 1 VCF file per chromosome requested.
+##
+## Inputs:
+## - intervals_file: GCS path of intervals file
+## - chromosomes: An array of chromosome to gather, such as ["1", "2", "21", "22", "X", "Y"]
+## - shards_to_ignore: An array of numbers from the intervals file to ignore, ex [576, 1032]
+## - output_suffix: Output file will look like "chr22${output_suffix}.vcf"
+## - input_directory: GCS path of directory containing sharded vcf files
+## - input_file_prefix: Input VCF files look like "${input_file_prefix}.n.${input_file_suffix}"
+## - input_file_suffix: Input VCF files look like "${input_file_prefix}.n.${input_file_suffix}"
+##
+## - VM configuration
+##   - Docker image url
+##   - VM disk size needed for chromosome 1
+##   - VM memory
+##   - Num CPU cores
+##   - Runtime zones, ex: "us-central1-a us-central1-b"
+##   - Number of times to try the workflow with a preemptible VM before
+##     falling back to a full-price VM.
+##
+## Outputs:
+## - Per chromosome 
+##   - <chromosome>.vcf.gz
```

Comment:
We should either remove the output_suffix or update this comment with the correct file name format.